pages/linux/dmidecode.md

```diff
@@ -15,10 +15,14 @@
 
 `sudo dmidecode -s system-serial-number`
 
-- Show all the BIOS information:
+- Show BIOS information:
 
 `sudo dmidecode -t bios`
 
-- Show all the system information:
+- Show CPU information:
 
-`sudo dmidecode -t system`
+`sudo dmidecode -t cpu`
```

Comment:
I'm not sure whether we have the different version of `dmidecode`.
But I do not have the `-t cpu` argument in my machine. only `dmidecode -t processor` available.

And below are the available `-t` arguments in my `dmidecode`:

```
➜  ~ dmidecode --version              
3.1

➜  ~ dmidecode -t

Valid type keywords are:
  bios
  system
  baseboard
  chassis
  processor
  memory
  cache
  connector
  slot
```

Reply:
@jsonbruce What OS and version are you using there?

Reply:
I get this too with Ubuntu 17.10 (dmidecode v3.1).

Reply:
Thanks ! updated.

Reply:
Confirmed! Thanks for updating, @agnivade